src/bullets/data_source/data_source_fmp.py

```diff
@@ -1,10 +1,11 @@
-from datetime import datetime, date, timedelta
 import json
 import math
-
+from datetime import datetime, date, timedelta
 from bullets.data_source.data_source_interface import DataSourceInterface, Resolution
 from bullets.data_source.recorded_data import *
 
+__all__ = ["FmpDataSource"]
```

Comment:
This is a general comment for `__all__` variables, since we use private variables and methods now (leading underscore), we don't really need __all__ variables anymore

Reply:
fixed